code/game/objects/items/grenades/flashbang.dm

```diff
@@ -34,7 +34,7 @@
 		living_mob.Knockdown(max(200/max(1, distance), 60))
 
 //Bang
-	if(!distance || loc == living_mob || loc == living_mob.loc) //Stop allahu akbarring rooms with this.
+	if(!distance || loc == living_mob || loc == living_mob.loc)
```

Comment:
this is the only change allowed to stay.

Reply:
I think some of the station prefixes are pretty questionable too, thoughts on those?